docs/IE_DG/Extensibility_DG/AddingNGraphOps.md

```diff
@@ -67,7 +69,7 @@ To add custom operations to the [Extension](Extension.md) class, create an opera
 
 This method returns a map of opsets that exist in the extension library.
 
-nGraph provides opsets mechanism for operation versioning. Different opsets distinguish between different versions of one operation.
+nGraph provides an opset mechanism for operation versioning. Different opsets distinguish between different versions of one operation.
```

Comment:
@ilyachur , the opset mechanism is not used for versioning of operations. It is used to group operations into some clusters. So I would recommend to refactor these sentences.

Reply:
Changed to `nGraph provides an opset mechanism to group operations into clusters.`